platforms/web/package.json

```diff
@@ -35,8 +35,7 @@
     "react-i18next": "^12.3.1",
     "react-router": "6.14.2",
     "react-router-dom": "6.14.2",
-    "wicg-inert": "^3.1.1",
-    "zustand": "^3.6.9"
```

Comment:
Do we remove it here because we already have it in `common`?

Reply:
`zustand` was only used in the AppRoutes for the profiles logic. After moving that logic to `ui-react` knip marked that dependency unused